sdk/monitor/azure-monitor-query/src/test/java/com/azure/monitor/query/MetricsClientTest.java

```diff
@@ -75,4 +75,33 @@ public void testMetricsBatchQueryDifferentResourceTypes() {
 
     }
 
+    @Test
+    public void testQueryResourcesReturnsNonNullResourceId() {
```

Comment:
Same here - we can just update the previous test to check for resource id in the response.